packages/amplify-graphql-api-construct/src/types.ts

```diff
@@ -427,6 +427,13 @@ export interface TranslationBehavior {
    */
   readonly useSubUsernameForDefaultIdentityClaim: boolean;
 
+  /**
+   * When disabled, relational fields on the mutation response are over-redacted to protect the data from subscribed users
+   * to correctly apply the auth rules.
```

Comment:
Suggest something like:

```
   * When enabled, suppresses default behavior of redacting relational fields when auth rules do not exactly match.
   * See <link to doc update>
```

Reply:
Updated the comment. Doc update isn't ready, will update once we have the docs update are published.